tests/publiccloud/migration.pm

```diff
@@ -37,10 +37,18 @@ sub run {
     record_info('installs SLE15-Migration and suse-migration-sle15-activation');
 
     $instance->run_ssh_command(cmd => "sudo zypper -n in SLES15-Migration suse-migration-sle15-activation");
+
+    # Upload distro_migration.log
+    $instance->upload_log("/system-root/var/log/distro_migration.log", failok => 1);
+
     record_info('system reboots');
     my ($shutdown_time, $startup_time) = $instance->softreboot(
         timeout => get_var('PUBLIC_CLOUD_REBOOT_TIMEOUT', 400)
     );
+
+    # Upload distro_migration.log
+    $instance->upload_log("/var/log/distro_migration.log", failok => 1);
```

Comment:
Both filenames are the same, I think they overwrite each other. Perhaps adding a custom `log_name` according to [this](https://open.qa/api/testapi/#_upload_logs) might help to avoid this?

Reply:
Ah ok, so I understand it as follows: Upload the pre-reboot log, do the reboot and overwrite it afterwards. If the reboot fails then the pre-reboot log is present and otherwise the post-reboot log is there.